src/lib/button-toggle/button-toggle.ts

```diff
@@ -264,7 +264,8 @@ export class MatButtonToggleGroup extends _MatButtonToggleGroupMixinBase impleme
     }
 
     if (this.multiple && Array.isArray(this._rawValue)) {
-      return !!this._rawValue.find(value => toggle.value != null && value === toggle.value);
+      const match = this._rawValue.find(value => toggle.value != null && value === toggle.value);
+      return typeof match !== 'undefined';
```

Comment:
You should be able to do `return match != null`, which will capture `null` and `undefined` without any of the other falsy values